rules/CodingStyle/Rector/ClassConst/VarConstantCommentRector.php

```diff
@@ -92,6 +96,10 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        if ($this->shouldSkipStringType($phpDocInfo, $node->consts[0]->value)) {
+            return null;
+        }
```

Comment:
This should be handled in string type mapper.

Reply:
ok, it previous got ` @param non-empty-string $expectedName` removed on used by other rule, I will check more.

Reply:
updated to check on IdentifierTypeMapper https://github.com/rectorphp/rector-src/pull/2451/commits/aee64c7b05c497ed4fe01923816c0fcc13570446

Reply:
checking in `IdentifierTypeMapper` seems cause removal of `@param` again:

```diff
-    /**
-     * @param non-empty-string $function
-     * @param non-empty-string $class
-     * @param non-empty-string $arrayMethod
-     * @param non-empty-string $nonArrayMethod
-     */
```

Reply:
Let's try register `AccessoryNonEmptyStringType` into `ScalarStringToTypeMapper` https://github.com/rectorphp/rector-src/pull/2451/commits/831a7e3dfd5c5ff308fc20330f513466036c2df6

Reply:
it works 🎉

Reply:
Thank you :